File: Python3/Tarifa.py
#https://open.kattis.com/problems/tarifa
limit = int(input())
size = int(input())
use = [None] * size
for i in range(size):
    use[i] = int(input())
final = (size + 1) * limit
for i in range(size):
    final -= use[i]
print(final)

File: C/Tarifa.c
//https://open.kattis.com/problems/tarifa
#include <stdio.h>
//#include <string.h>
//#include <stdlib.h>

int main() {
	int x, n, temp;
	scanf("%d", &x);
	scanf("%d", &n);
	int sum = (n+1) * x;
	for (int i = 0; i < n; i++)
	{
		scanf("%d", &temp);
		sum -= temp;
	}
	printf("%d", sum);
  return 0;
}
